contrib/samples/seldon/iris_storagebucket.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import json
+import kfp.dsl as dsl
+from string import Template
+
+@dsl.pipeline(
+    name="Deploy example SKLearn Iris",
+    description="SKLearn Iris simple deployment example"
+)
+def iris_storagebucket(bucket='gs://seldon-models/sklearn/iris'):
+
+#simple serving of an iris sklearn model based on https://docs.seldon.io/projects/seldon-core/en/latest/servers/overview.html
+#requires seldon 0.3.2 or higher
+    sklearnjsonTemplate = Template("""
```

Comment:
Is there a specific reason to use camel case naming convention here?
```suggestion
    sklearnjson_template = Template("""
```

Reply:
Same comment to other occurrences.

Reply:
Good point, I've applied this to all the variable names